package.json

```diff
@@ -2,7 +2,7 @@
   "name": "tonomy-app-websites",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
```

Comment:
this is a major change. please investigate potential consequences. I've seen many things go wrong with this.

Reply:
after adding ENABLE_EXPERIMENTAL_COREPACK environment variable getting this error https://vercel.com/tonomy-foundation/tonomy-app-websites/G26FFRBNHDDY4AGcmECBYLmh7GQ5 and i get this change from google